### PR TITLE
Tree for parsing

### PR DESCRIPTION
**Pull request by shivupa**
This is the tree implementation for cclib2 parsing

### CODE REVIEW

File: cclib/notes.md
Comment (amandadumi) [inline cclib/notes.md:1]: is this an intentional add?

Comment (shivupa) [reply cclib/notes.md:1]: oops

File: cclib/tree/tree.py
Comment (amandadumi) [inline cclib/tree/tree.py:1]: would it be safe to remove commented code or is that there for reference as we develop?

**Review by amandadumi — APPROVED** [review]
perfect, thanks!

**Pull request merged**